airgun/views/computeresource.py

```diff
@@ -112,3 +114,28 @@ def after_fill(self, was_change):
     def is_displayed(self):
         return self.browser.wait_for_element(
             self.name, exception=False) is not None
+
+
+class ResourceProviderDetailView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and self.breadcrumb.locations[0] == 'Compute resources'
+                and not self.breadcrumb.read().starts_with('Edit ')
```

Comment:
Maybe there's smth a bit better to count on, like `details` word in current location or smth similar? If not - then, at least, add one more condition `and not self.breadcrumb.read() == 'Create Compute Resource'` but we'd rather specify what should be there instead everything that shouldn't :)